### PR TITLE
HOCS-6434 Initialise repo

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**
Repo initialisation with running workflows and basic directory structure

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+on:
+  pull_request:
+    types: [ opened, reopened, synchronize ]
+
+jobs:
+  test:
+    uses: UKHomeOffice/sas-github-workflows/.github/workflows/test-gradle.yml@v1
+    with:
+      dockerComposeComponents: 'localstack postgres'
+      dockerComposeCommand: './ci/docker-compose.yml -f ./ci/docker-compose.elastic.yml'
```
Comment (LiamMacP) [inline .github/workflows/test.yml:11]:
```suggestion
      dockerComposeCommand: './ci/docker-compose.yml'
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+For versioning this project uses SemVer.
+
+## Authors
+
+This project is authored by the Home Office.
+
+## License
+
+This project is licensed under the MIT license. For details please see [License](LICENSE)
```
Comment (LiamMacP) [inline README.md:16]: We should bring across the License file?

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+dependencies {
+    annotationProcessor 'org.projectlombok:lombok'
+    annotationProcessor 'org.springframework:spring-context-indexer:5.3.23'
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+    compileOnly 'org.projectlombok:lombok'
+    developmentOnly 'org.springframework.boot:spring-boot-devtools'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-json'
+    implementation 'com.vladmihalcea:hibernate-types-55:2.21.1'
```
Comment (LiamMacP) [inline build.gradle:28]: Worth just pulling out this and guava for now as a nice basline?

File: README.md (hunk 2 above)
Comment (robertdeniszczyc2) [reply README.md:16]: It was already added when the repo was created: https://github.com/UKHomeOffice/hocs-cdlz-document-extractor/blob/main/LICENSE

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**